lib/lambda_ethereum_consensus/logger/console_logger.ex

```diff
@@ -0,0 +1,46 @@
+defmodule ConsoleLogger do
+  @moduledoc """
+  Custom logger formatter for console output.
+  """
+
+  def format(level, message, timestamp, metadata) do
+    pattern = Logger.Formatter.compile(" $time $message ")
+
+    formatted_metadata =
+      Enum.map_join(
+        metadata,
+        " ",
+        fn {key, value} ->
+          IO.ANSI.bright() <>
```

Comment:
We should disable colors when logging to a file (we can check with `IO.ANSI.enabled?()`)

Reply:
But this is just the console logger though

Reply:
Yes. But it's the one used when piping data to a file (i.e. when doing `mix run > some_file.txt`).

Reply:
Sure! In fact, [according to docs](https://hexdocs.pm/logger/1.16.0/Logger.html#module-backends-and-backwards-compatibility), it's the right way to do it.